### ISSUE TITLE
LND won't start (v1.11)

### ISSUE THREAD

**Issue by Godrigos** [raspiblitz/raspiblitz#4740]
**Describe the bug**

A few days ago my raspiblitz (1.11) stop working properly. It now gets stuck in the "Node Syncing" screen with blockchain progress at 100% (several peers) and Lightning progress at prepeare sync, EMPTY, or a single time 100% but 0 peers !?...

The point is, I cant't reach SSH Menu to do something...

I don't know how to get logs, would appreciate some orientation to provide better debugging.
And the most important! Is there a way to at least recover funds?

Thanks for the attention.

UPDATE:
I get this error trying to use lncli.
₿ lncli getinfo
[lncli] rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing: dial tcp 127.0.0.1:10009: connect: connection refused"

**Comment by openoms** [comment]
press CTRL+C ro get into the prompt at the terminal.
Type `menu` for the menu.
`blitzhelp` for help


`debug -l` to share your debug logs link.

**Comment by Godrigos** [comment]
Thanks for reaching out, but once I was already thinking of changind my ssd and this happened. I just created another installation with version 1.11.2 and used SEED + SCB to recover. Unfortunatly I wasn't able to copy the blockchain (errors while trying to copy), so it is downloadind again.
Therefore, I won't be able to give you debugging information on this anymore, just hoping to have my funds back when the sync finishes.

**Comment by Godrigos** [comment]
Hi there, me again.
I will post here since it's a continuation of the same issue.
As said above I installed the newer version of raspiblitz in a new sd card. And entered SEED + SCB.
All seemed to be going well, the blockchain was downloading again...
Well, about a few days ago (blockchain was about 87%) I supose the raspiblitz restarted by its own (asking for LN wallet password) and from that day on I only get:

```
Blockcain progress: wainting for start
Lightining progress: prepare sync
```

I already restarted and rebooted, this is all that I get.

P.s.: The rasp seems to have normal internet access, since I can ping sites from it.

[Here](https://termbin.com/r8bq) are the logs.

Any ideas?
Thanks for the help and patience.
